src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.builds

```diff
@@ -6,28 +6,12 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.Cng.csproj">
-      <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard1.6</TargetGroup>
```

Comment:
@joperezr Nope, the fixes I'd be concerned about were made fairly early in the 1.1 timeframe. I certainly haven't heard any notion that we'd do things on release branches (except for major bugs); but if that's really the plan, that's fine, I guess.